app/controllers/profile_controller.rb

```diff
@@ -31,6 +31,8 @@ def update
     redirect_to controller: :profile, action: :edit
   end
 
+  # :nocov:
+
```

Comment:
Please remove this nocov

Reply:
this is used for private method because as per "Rails Conf 2013 The Magic Tricks of Testing by Sandi Metz" taught in the coursework, we should not be testing private methods explicitly. And hence it should also not be included in the total coverage.